.golangci.yaml

```diff
@@ -59,6 +60,12 @@ linters-settings:
       WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
       See the License for the specific language governing permissions and
       limitations under the License.
+  gci:
+    sections:
+      - standard # Standard section: captures all standard packages.
+      - default # Default section: contains all imports that could not be matched to another section type.
```

Comment:
Should we add "dot" here as a section since we have a number of dot imports that we should probably group together throughout the testing code?

Reply:
cc @jonathan-innis @ellistarn

Reply:
cc @jonathan-innis @ellistarn @njtran 
Please take a look, thanks

Reply:
cc @jonathan-innis